app/src/main/AndroidManifest.xml

```diff
@@ -9,6 +9,10 @@
     -->
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION"/>
     <uses-permission android:name="android.permission.INTERNET"/>
+    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"/>
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
+    <uses-feature android:name="android.hardware.camera"
+        android:required="true" />
```

Comment:
Having this as required means that anybody who doesn't have a camera won't be able to install the app. Is that something we want? I think you can set it to false and then just check if the phone has a camera whenever you want to use a camera feature, then we don't have this issue.